pkgs/top-level/python-packages.nix

```diff
@@ -10065,6 +10065,27 @@ in {
     };
   };
 
+  OWSLib = buildPythonPackage rec {
+    pname = "OWSLib";
+    version = "0.16.0";
+    name = "${pname}-${version}";
```

Comment:
The `name` attribute is added by `buildPython*` and should therefore be removed.